src/ring/logger/messages.clj

```diff
@@ -26,7 +26,10 @@
 (defn- make-default-starting-message
   [options {:keys [request-method uri remote-addr query-string headers] :as req}]
   (str request-method " "
-       uri (if query-string (str "?" query-string))
+       (if (and query-string
+                (:log-query-string? options)) 
+             (str uri "?" query-string)
+             uri)
```

Comment:
A minor thing: As we are here, could you please extract `uri` out from the conditional into a separate line, and change the conditional to something like

```clojure
(when (and query-string (:log-query-string? options)
   (str "?" query-string))`
```

Thanks!

Reply:
Ok, changed it :)